setup.py

```diff
@@ -109,7 +109,8 @@ def compile_cuda_module(host_args):
     cuda_home, nvcc = get_cuda_path()
 
     print("NVCC ==> ", nvcc)
-    arch_flags = "-arch=sm_60 " + \
+    arch_flags = "-arch=sm_37 " + \
```

Comment:
Is there a disadvantage to adding more architectures here?  Is it possible to add many more architectures - or perhaps a better way may be to also allow the user to configure this somehow to pass their architecture type, or have some way to automatically detect/figure out the architecture based on the current machine?